internal/json/sonic.go

```diff
@@ -0,0 +1,13 @@
+//go:build sonic && amd64 && avx512 && !stdjson
+// +build sonic,amd64,avx512,!stdjson
+
+package json
+
+import "github.com/bytedance/sonic"
+
+var Marshal = sonic.ConfigStd.Marshal
+var MarshalIndent = sonic.ConfigStd.MarshalIndent
```

Comment:
I guess since it's internal package, there is no need of defining functions that are not used in lib, please remove unused